{{ cookiecutter.package_name|slugify }}/docker-compose.yml

```diff
@@ -43,15 +39,15 @@ services:
     build:
       context: .
       target: app
-      {%- if cookiecutter.private_package_repository_name %}
-      secrets:
-        - poetry_auth
-      {%- endif %}
     tty: true
     {%- if cookiecutter.with_fastapi_api|int or cookiecutter.with_streamlit_app|int %}
     ports:
       - "8000:8000"
     {%- endif %}
+    {%- if cookiecutter.private_package_repository_name %}
+    secrets:
+      - poetry_auth
+    {%- endif %}
```

Comment:
These are run-time secrets, which I don't think are necessary for the app image since you normally won't need to install additional dependencies in the app image at run-time. But let me know if I'm missing something!